filenames.go

```diff
@@ -87,7 +87,7 @@ func AbsoluteFilename(dt DocType, typ string, index int) string {
 			Log("unsupported type %s pair and %v", typ, dt)
 		}
 
-	case ThemeType, ThemeContentType:
+	case ThemeType, ThemeTypeStrict, ThemeContentType:
```

Comment:
Is there no ThemeContentTypeStrict ?  I notice that sometimes there are both non-strict and strict (2 types) and sometimes there are 3 with 2 non strict and 1 strict matching only one of the nonstrict types: e.g. ThemeType - ThemeTypeStrict, ThemeContentType